### PR TITLE
Fix/scripts to deploy new contracts

### LINKED ISSUE #371: incorrect deployment scripts
inside contracts/scripts the scripts are referencing bad contract and wrong url e.g. 
- dollar/dollarMintingCalculator.sol instead of src/dollar/core/dollarMintCalculator.sol
- BondingShareV2.sol etc... all these contracts have been remloved/renamed

### PR DESCRIPTION
**Pull request by zgorizzo69**
Resolves #371 

you can once again follow the readme and deploy with  `yarn workspace @ubiquity/contracts deploy CreditNFT --manager 0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98 --network mainnet`

### CODE REVIEW

File: packages/contracts/scripts/deploy/manager.ts [hunk 1]
```diff
@@ -1,102 +1,119 @@
...
-    handler: ubiquityFormulaFunc,
-    options: ubiquityFormulaOptions,
+  UbiquityDollarToken: {
+    handler: ubiquityDollarTokenFunc,
+    options: ubiquityDollarTokenOptions,
   },
-  YieldProxy: {
-    handler: yieldProxyFunc,
-    options: yieldProxyOptions,
+  UbiquityGovernanceTokenFunc: {
```
Comment (0x4007) [inline packages/contracts/scripts/deploy/manager.ts]: The naming convention here is incorrect. This is likely supposed to be `governance`. Same with the others.

Comment (0x4007) [reply packages/contracts/scripts/deploy/manager.ts]: Update: just reviewed https://github.com/ubiquity/ubiquity-dollar/issues/292#issuecomment-1288255782 and this should be `UbiquityGovernanceToken`

Comment (zgorizzo69) [reply packages/contracts/scripts/deploy/manager.ts]: good cacth fixed !

File: packages/dapp/components/lib/hooks/contracts/useManagerManaged.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ async function connectManagerContracts(manager: Contract, provider: NonNullable<
     dollarToken: getUbiquityAlgorithmicDollarContract(dollarToken, provider),
     dollarMetapool: getIMetaPoolContract(dollar3poolMarket, provider),
     dollarTwapOracle: getTWAPOracleContract(twapOracle, provider),
-    dollarMintingCalculator: getDollarMintingCalculatorContract(dollarMintCalc, provider),
+    dollarMintingCalculator: getDollarMintCalculatorContract(dollarMintCalc, provider),
```
Comment (0x4007) [inline packages/dapp/components/lib/hooks/contracts/useManagerManaged.tsx]:
```suggestion
    dollarMintCalculator: getDollarMintCalculatorContract(dollarMintCalc, provider),
```

File: packages/contracts/scripts/deploy/manager.ts [hunk 3]
```diff
@@ -1,102 +1,119 @@
...
-    handler: bondingShareV2Func,
-    options: bondingShareV2Options,
+  StakingShare: {
+    handler: stakingShareFunc,
+    options: stakingShareOptions,
   },
-  CouponsForDollarsCalculator: {
-    handler: couponsForDollarsCalculatorFunc,
-    options: couponsForDollarsCalculatorOptions,
+  ubiquityChef: {
```
Comment (0x4007) [inline packages/contracts/scripts/deploy/manager.ts]: Why is this the only one not capitalized?

**Review by 0x4007 — CHANGES_REQUESTED** [review]

File: packages/contracts/scripts/deploy/manager.ts (hunk 3 above)
Comment (0x4007) [reply packages/contracts/scripts/deploy/manager.ts]:
```suggestion
  UbiquityChef: {
```

Comment (zgorizzo69) [reply packages/contracts/scripts/deploy/manager.ts]: > 

it is done but it seems that for GH it is still requested

**Pull request merged**